core/interfaces/status.py

```diff
@@ -235,7 +256,7 @@ class GsStatusOpenFileCommand(TextCommand, GitCommand):
 
     def run(self, edit):
         lines = util.view.get_lines_from_regions(self.view, self.view.sel())
-        file_paths = (line.strip() for line in lines if line[:4] == "    ")
+        file_paths = (line[5:].strip() for line in lines if line[5:8] == "   ")
```

Comment:
I feel we should take this opportunity to clarify what these slices mean and, possibly, replace with some sort of regex/syntax-scope matching instead?

Reply:
yes, especially as I got it wrong... it should be `line[7:].strip()`

Reply:
it seems to break too when a file was modified 100 weeks ago.